examples/platform/silabs/MatterConfig.cpp

```diff
@@ -331,6 +331,7 @@ CHIP_ERROR SilabsMatterConfig::InitWiFi(void)
 extern "C" void vApplicationIdleHook(void)
 {
 #if (SLI_SI91X_MCU_INTERFACE && CHIP_CONFIG_ENABLE_ICD_SERVER)
-    sl_si91x_invoke_btn_press_event();
+    sl_si91x_btn_event_handler();
+    sl_si91x_uart_power_requirement_handler();
```

Comment:
Can you provide information on this `sl_si91x_uart_power_requirement_handler` It is not in the pr description and I don't see other code changes related to uart.

Reply:
@jmartinez-silabs , this is not a new function, the functionality of UART was also coded into the same function as BTN event handler, I have just decoupled the activities and named them.